### PR TITLE
Fix UI build in fresh repo

### LINKED ISSUE #7863: v.1.2.4 fails to build from source on FreeBSD
**Describe the bug**

On FreeBSD vault is compiled from source, using build instructions from hashicorp repo.
Currently this fails during `static-assets` step due to missing `pkg/web_ui` dir

```
$ git clean -fdx && git reset --hard
HEAD is now at 360b3a32c release: stage v1.2.4

$ go version
go version go1.13.4 freebsd/amd64

$ env GOPATH=${PWD} PATH=${PWD}/bin:${PATH} gmake bootstrap static-assets static-dist
Installing/Updating golang.org/x/tools/cmd/goimports
Installing/Updating github.com/elazarl/go-bindata-assetfs/...
Installing/Updating github.com/hashicorp/go-bindata/...
Installing/Updating github.com/mitchellh/gox
Installing/Updating github.com/kardianos/govendor
Installing/Updating github.com/client9/misspell/cmd/misspell
Installing/Updating github.com/golangci/golangci-lint/cmd/golangci-lint
--> Generating static assets
bindata: Failed to stat input path 'pkg/web_ui': lstat pkg/web_ui: no such file or directory
Error: go-bindata:  exit status 1
gmake: *** [Makefile:141: static-assets] Error 1

```

**To Reproduce**

```
$ sh
$ cd /tmp/
$ curl -#L https://github.com/hashicorp/vault/archive/v1.2.4.tar.gz | tar xzf -
$ cd vault-1.2.4
$ env GOPATH=${PWD} PATH=${PWD}/bin:${PATH} gmake bootstrap static-assets static-dist
```

* Server Operating System/Architecture:

- FreeBSD 12.1R with Go 1.13.4 compiler

**Additional context**

This presumably is broken for anybody building from git or source tarball at present - see https://github.com/hashicorp/vault/issues/7701 for a similar but not identical issue.

### PR DESCRIPTION
**Pull request by swills**
Fixes #7863

### CODE REVIEW

**Comment by swills** [comment]
Can anyone tell me if these CI failures are related to this change or normal failures?

**Comment by dch** [comment]
+1 "works for me" too. many thanks!

File: Makefile [hunk 1]
```diff
@@ -17,7 +17,7 @@ EXTERNAL_TOOLS=\
 	github.com/kardianos/govendor \
 	github.com/client9/misspell/cmd/misspell \
 	github.com/golangci/golangci-lint/cmd/golangci-lint
-GOFMT_FILES?=$$(find . -name '*.go' | grep -v pb.go | grep -v vendor)
+GOFMT_FILES?=$$(find . -name '*.go' | grep -v pb.go | grep -v vendor | grep -v testdata)
```
Comment (catsby) [inline Makefile]: Can you detail why this change is necessary?

Comment (swills) [reply Makefile]: goimports -w $(find . -name '*.go' | grep -v pb.go | grep -v vendor)
./src/github.com/golangci/golangci-lint/test/testdata/notcompiles/typecheck.go:4:1: expected declaration, found fun
./src/golang.org/x/tools/internal/lsp/testdata/channel/channel.go:23:14: expected operand, found ')'
./src/golang.org/x/tools/internal/lsp/testdata/channel/channel.go:25:3: missing ',' in argument list
./src/golang.org/x/tools/internal/lsp/testdata/importedcomplit/imported_complit.go:25:22: expected selector or type assertion, found '}'
./src/golang.org/x/tools/internal/lsp/testdata/importedcomplit/imported_complit.go:26:3: expected '}', found 'EOF'
gmake[1]: *** [Makefile:208: fmt] Error 2

**Comment by swills** [comment]
ping

File: Makefile (hunk 1 above)
Comment (catsby) [reply Makefile]: I believe this is due to the way you're invoking `gmake` and setting `GOPATH` to the current working directory, would you agree?

```
env GOPATH=${PWD} PATH=${PWD}/bin:${PATH} gmake ...
```

Comment (swills) [reply Makefile]: Hmm, even when only calling it as `env PATH=${PWD}/bin:${PATH} gmake bootstrap static-dist dev-ui static-assets` I still see the errors.

Comment (swills) [reply Makefile]: (If I remove the `PATH`, I get `gmake: go-bindata-assetfs: Command not found`)

**Comment by swills** [comment]
ping

File: Makefile (hunk 1 above)
Comment (swills) [reply Makefile]: Any update?

Comment (briankassouf) [reply Makefile]: I believe you can get around this by setting your GOPATH and PATH variables to a different directory. Could you undo this change so we can get the below `static-assets` fix merged in? If there are still issues the the fmt call a new PR can be opened.

Comment (catsby) [reply Makefile]: I opened https://github.com/hashicorp/vault/pull/7979 to revert this particular part of the change.

Comment (swills) [reply Makefile]: What should GOPATH and PATH be set to?

Comment (swills) [reply Makefile]: Nevermind, doesn't seem necessary.
